presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java

```diff
@@ -1609,7 +1826,7 @@ private static double computeSphericalExcess(Polygon polygon, int start, int end
             calculator.add(point);
         }
 
-        return calculator.computeSphericalExcess();
+        return calculator;
     }
 
     private static class SphericalExcessCalculator
```

Comment:
SphericalExcessCalculator calculator class evolved and it now computes multiple things, not just spherical excess. I suggest to rename it to SphericalCalculator and, perhaps, move into a separate file. 

I further suggest to move `initializeSphericalExcessCalculator` method into  SphericalCalculator class and rename it to `initialize`.

I then suggest to make SphericalCalculator's constructor private and add 3 static methods:

```
        public static double computeSphericalExcess(Polygon polygon, int start, int end)
        {
            return initialize(polygon, start, end).computeSphericalExcess();
        }

        public static boolean containsPole(Polygon polygon, int start, int end)
        {
            return initialize(polygon, start, end).containsPole();
        }

        public static double computeExtrapolatedLatitude(Point start, Point end, double longitude)
        {
            return new SphericalCalculator(start).computeExtrapolatedLatitude(end, longitude);
        }
```